packages/palette_generator/lib/palette_generator.dart

```diff
@@ -678,10 +665,10 @@ class PaletteColor with Diagnosticable {
       // the same lightness, we need to use mismatched values
       _bodyTextColor = lightBodyAlpha != null //
           ? white.withAlpha(lightBodyAlpha)
-          : black.withAlpha(darkBodyAlpha);
+          : black.withAlpha(darkBodyAlpha ?? 0);
```

Comment:
The fallback might be better as 255 instead of 0: zero will make the colors entirely invisible, which isn't a useful value. 255 might be ugly, but at least it is possible for it to be seen.

Reply:
Makes sense 👍 Colors are kinda a new universe for me. 😃